lib/Browser.js

```diff
@@ -33,6 +33,9 @@ class Browser extends EventEmitter {
     this._closeCallback = closeCallback || new Function();
     /** @type {Map<string, Target>} */
     this._targets = new Map();
+    this._connection.on('ws.closed', closed => {
```

Comment:
we shouldn't whitelist certain events and give them special treatment.

Instead, let's introduce a "closedCallback" for the connection and initialize it here:

```js
this._connection.setClosedCallback(() => {
  this.emit(Browser.Events.Disconnected);
});
```